scss/tech.scss

```diff
@@ -0,0 +1,53 @@
+$tech-logo-size: 100px;
+$tech-span-width: 200px;
+$tech-span-height: 300px;
+$max-percentaje: 100%;
+$border-radius: 10px;
```

Comment:
I'm not sure how usefull this is, are you gonna use this same border radius in many classes?